### ISSUE TITLE
ModuleNotFoundError: No module named 'oomox_gui.i18n'

### ISSUE THREAD

**Issue by ivanbartsov** [themix-project/oomox#158]
```
Traceback (most recent call last):
  File "/usr/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/opt/oomox/oomox_gui/__main__.py", line 1, in <module>
    from .main import main
  File "/opt/oomox/oomox_gui/main.py", line 18, in <module>
    from .theme_file_parse import read_colorscheme_from_path
  File "/opt/oomox/oomox_gui/theme_file_parse.py", line 1, in <module>
    from .theme_model import THEME_MODEL
  File "/opt/oomox/oomox_gui/theme_model.py", line 4, in <module>
    from .plugin_loader import (
  File "/opt/oomox/oomox_gui/plugin_loader.py", line 46, in <module>
    init_plugins()
  File "/opt/oomox/oomox_gui/plugin_loader.py", line 29, in init_plugins
    os.path.join(plugin_path, "oomox_plugin.py")
  File "/opt/oomox/oomox_gui/helpers.py", line 23, in get_plugin_module
    spec.loader.exec_module(module)
  File "/opt/oomox/oomox_gui/../plugins/oomoxify/oomox_plugin.py", line 8, in <module>
    from oomox_gui.i18n import _
ModuleNotFoundError: No module named 'oomox_gui.i18n'
```

Platform: Arch Linux
Version: oomox 1.7.0.2-3 from[ the AUR](https://aur.archlinux.org/packages/oomox)

Blame led me to [this commit](https://github.com/themix-project/oomox/commit/f7d48b586e3cd7bf93ef9ca5339353eabd16c1b4), but I may be mistaken

**Comment by actionless** [comment]
is it fine with `oomox-git`?

**Comment by actionless** [comment]
the commit which you are referencing is newer than version 1.7.0.2

**Comment by actionless** [comment]
i've updated aur package for `oomox`, please try now

**Comment by ivanbartsov** [comment]
> i've updated aur package for oomox, please try now

Yeah, now the error is gone, thanks for the quick reaction! :+1: 

> is it fine with oomox-git?

Yup, oomox-git is ok.

Also, now I noticed with `oomox`  the `/opt/oomox/plugins/import_base16/base16-data/` folder is empty (so oomox logs an error and the base16 preset group is empty in the UI). I checked the PKGBUILD files and the one in the repo, used for `oomox-git` has `base16-builder` repo listed as a submodule, while PKGBUILD for `oomox` doesn't reference the repo at all.

Let me know if you want me to file a separate issue for that.
